sqlmesh/core/test/definition.py

```diff
@@ -60,6 +66,8 @@ def __init__(
         preserve_fixtures: bool = False,
         default_catalog: str | None = None,
         concurrency: bool = False,
+        verbosity: Verbosity = Verbosity.DEFAULT,
+        rich_output: bool = True,
```

Comment:
Why do we need this option and not just default to rich output all the time?

Reply:
Doesn't seem like a good argument to keep it. We should instead update the tests, imo, unless there's a compelling reason to expose a non-rich formatting option.

How does this look in formats like CI/CD bot, magics, etc?